src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ChainPal.cs

```diff
@@ -25,7 +25,26 @@ public static bool ReleaseSafeX509ChainHandle(IntPtr handle)
             DateTime verificationTime,
             TimeSpan timeout)
         {
-            return new OpenSslX509ChainProcessor();
+            CheckRevocationMode(revocationMode);
+
+            X509Certificate2 leaf = new X509Certificate2(cert.Handle);
+
+            X509Certificate2Collection candidates = OpenSslX509ChainProcessor.FindCandidates(leaf, extraStore);
+
+            return OpenSslX509ChainProcessor.BuildChain(
+                leaf,
+                candidates,
+                applicationPolicy,
+                certificatePolicy,
+                verificationTime);
+        }
+
+        private static void CheckRevocationMode(X509RevocationMode revocationMode)
+        {
+            if (revocationMode != X509RevocationMode.NoCheck)
+            {
+                throw new NotImplementedException(SR.WorkInProgress);
```

Comment:
Could we add comments for such NotImplementedExceptions calling out why they're not implemented, e.g. ```// TODO: Implement once a stable networking stack is available"?  Presumably the only NotImplementedExceptions we have are ones with TODOs associated with them, as otherwise they'd be promoted to NotSupportedException or PlatformNotSupportedException.  If there was a GitHub issue number to go along with the TODO, even better.